util/topgen.py

```diff
@@ -630,7 +636,7 @@ def create_mem(item, addrsep, regwidth):
                          swaccess=swaccess)
 
 
-def generete_rust(topname, completecfg, name_to_block, out_path, version_stamp,
+def generate_rust(topname, completecfg, name_to_block, out_path, version_stamp,
                   src_tree_top, topgen_template_path):
     # Template render helper
     def render_template(template_path: str, rendered_path: Path, **other_info):
```

Comment:
Since this PR is not touching that particular code, I probably want to hold off with that. I created an issue with that instead: https://github.com/lowRISC/opentitan/issues/24822

Reply:
Okay